plugins/source/aws/resources/services/fsx/file_caches_fetch.go

```diff
@@ -23,7 +22,7 @@ func fetchFsxFileCaches(ctx context.Context, meta schema.ClientMeta, parent *sch
 		if err != nil {
 			var ae smithy.APIError
 			if errors.As(err, &ae) && ae.ErrorCode() == "BadRequest" && ae.ErrorMessage() == "The requested feature is not enabled for this AWS account." {
-				log.Warn().Err(err).Msg("skipping resource")
+				cl.Logger().Warn().Err(err).Msg("skipping resource")
```

Comment:
maybe just returning the error? this way it will have consistent logging from the SDK?

Reply:
Done @yevgenypats